sites/packworld/server/templates/website-section/index.marko

```diff
@@ -36,12 +36,14 @@ $ const { id, alias, name, pageNode } = data;
   <@below-page>
     <marko-web-resolve-page|{ data: section }| node=pageNode>
       $ const aliases = hierarchyAliases(section);
+      $ const nativeX = { index: 1, name: "load-more" };
```

Comment:
Change optionId above to optionName, add another content flow here to prevent the load more from firing an immediate page view on load. Reference base-cms-websites/industrial-media#16